hr_employee_private_wizard/wizard/hr_employee_private_wizard.py

```diff
@@ -0,0 +1,103 @@
+# © 2020 - Today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from lxml import etree
+from odoo import api, models, fields, _
+
+WIZARD_FOOTER = """
+    <footer>
+        <button class="oe_highlight" name="save_and_close" type="object"/>
+        <button name="cancel" special="cancel"/>
+    </footer>
+"""
+
+
+class EmployeePrivateWizard(models.TransientModel):
+
+    _name = "hr.employee.private.wizard"
+    _description = "Employee Private Data Wizard"
+
+    employee_id = fields.Many2one("hr.employee")
+
+    def save_and_close(self):
+        return {'type': 'ir.actions.act_window_close'}
+
+    @api.multi
+    def read(self, fields=None, load="_classic_read"):
+        if self._context.get("read_wizard_data"):
+            return super().read(fields, load)
+
+        res = self._get_employee().read(fields, load)
+        return [dict(res[0], id=self.id)]
+
+    def _get_employee(self):
+        return (
+            self.with_context(read_wizard_data=True, lang=self.env.user.lang)
+            .sudo()
+            .employee_id
+        )
+
+    @api.multi
+    def write(self, vals):
+        self._get_employee().write(vals)
+        return {"type": "ir.actions.act_window_close"}
+
+    @api.model
+    def fields_view_get(self, *args, **kwargs):
+        result = (
+            self.env["hr.employee"].sudo().fields_view_get(view_type="form")
+        )
+        tree = etree.fromstring(result["arch"])
+        self._remove_non_private_arch_nodes(tree)
+        _make_name_field_readonly(tree)
+        _remove_image_field(tree)
+        tree.append(self._get_form_footer())
+        result["arch"] = etree.tostring(tree)
+        result["model"] = self._name
+        return result
+
+    def _get_form_footer(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-hr-addons/pullRequest?prid=5358386)